src/Tests/Home.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Great work, you wrote jest snapshot for `Home` and `Quote` pages but we need to write for `Calculator` page as well.

![image](https://user-images.githubusercontent.com/31897434/130496176-6acbced1-5b49-4385-b3fb-6c8d26467179.png)

Reply:
Thank you for the review, I've done it 👍